01-path-basics/103-kubernetes-concepts/readme.adoc

```diff
@@ -14,12 +14,12 @@ Kubernetes as a platform has a number of abstractions that map to API objects. T
 
 == Prerequisites
 
-This chapter uses a cluster with 3 master nodes and 5 worker nodes as described link:../102-your-first-cluster#multi-master[here].
+This chapter uses an EKS cluster with 3 worker nodes as described link:../102-your-first-cluster[here].
```

Comment:
Lets not specify the number of worker nodes as that might change late. Just pointing to the instructions is sufficient.

Reply:
Removing the 3 in a future commit.